ql/src/test/queries/clientnegative/compute_stats_long.q

```diff
@@ -3,5 +3,5 @@ create table tab_int(a int);
 -- insert some data
 LOAD DATA LOCAL INPATH "../../data/files/int.txt" INTO TABLE tab_int;
 
--- compute stats should raise an error since the number of bit vectors > 1024
-select compute_stats(a, 10000) from tab_int;
+-- this should raise an error since the number of bit vectors has a hard limit at 1024
+select compute_stats(a, 'fm', 10000) from tab_int;
```

Comment:
does this limit apply only to fm algo?

Reply:
yes see https://github.com/apache/hive/blob/5d735c20ffb6a8624382155a3e90e650db2fb80e/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/common/ndv/NumDistinctValueEstimatorFactory.java#L64

Reply:
thanks

Reply:
+1